buildconfig.js

```diff
@@ -1,19 +1,26 @@
 export default {
```

Comment:
what is this file for?

Reply:
It's a hack for database schema that we're keeping so we can change variables while doing dev work. It'll also make it easy for us to implement the schema in production. Uncomment and run the `fireDb.updateConfig` function to update it. Do you have suggestions to do this neater?

Reply:
Is the data in this file (ie `80%`) pulled in locally or is it example data? Is the schema enforced somehow?

Reply:
When the `fireDb.updateConfig` in index.js and the respective data in buildconfig.js are uncommented. The data would get pushed to firebase. So the data in this file are staging data. What do you mean by if the schema is enforced?

Reply:
The data is pushed to Firebase and getStaticProps pulls data from Firebase right after.

Reply:
Ah I see thanks, was wondering if the schema forces future data to be in a certain format/hierarchy. Guess not